orchprovider/registry.go

```diff
@@ -12,6 +13,7 @@ import (
 	"github.com/openebs/maya/types/v1"
 )
 
+// OrchProviderFactory type is used as provider's instance creating function
```

Comment:
@AmitKumarDas 
Can you please give a little insight here on the comment.

Reply:
`OrchProviderFactoryis the factory implementation of OrchProvider`